pkg/deviceplugin/manager.go

```diff
@@ -22,6 +22,10 @@ import (
 	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
 )
 
+type postAllocateFunc func(*pluginapi.AllocateResponse) error
+type preStartContainerFunc func(*pluginapi.PreStartContainerRequest) error
+type preferredAllocationFunc func(*pluginapi.PreferredAllocationRequest) (*pluginapi.PreferredAllocationResponse, error)
```

Comment:
I'd prefer to rename `preferredAllocationFunc` to `getPreferredAllocationFunc` for consistency.

Also all the other variables, parameters and struct fields `preferredAllocation` should be better renamed to become a verb to signal they contain a func, that is to `getPreferedAllocation`.

Reply:
I would not have preferred to prefix every preferred with get since not even the pluginapi gets it everywhere, but I get it, preferred or not.

Reply:
I agree. The interface name doesn't need to be prefixed indeed. Could you please revert it?

Reply:
I would prefer I got the gets before preferred preferably right now